backend/module/sourcecontrol/sourcecontrol.go

```diff
@@ -31,22 +42,104 @@ func New(*any.Any, *zap.Logger, tally.Scope) (module.Module, error) {
 		return nil, errors.New("service was not the correct type")
 	}
 
-	return &mod{c}, nil
+	return &mod{c, config}, nil
 }
 
 type mod struct {
 	github github.Client
+	config *sourcecontrolconfigv1.Config
 }
 
 func (m *mod) Register(r module.Registrar) error {
 	sourcecontrolv1.RegisterSourceControlAPIServer(r.GRPCServer(), m)
 	return r.RegisterJSONGateway(sourcecontrolv1.RegisterSourceControlAPIHandler)
 }
 
-func (m *mod) CreateRepository(ctx context.Context, req *sourcecontrolv1.CreateRepositoryRequest) (*sourcecontrolv1.CreateRepositoryResponse, error) {
-	return m.github.CreateRepository(ctx, req)
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func (m *mod) ManageableOrganizations(ctx context.Context) ([]*sourcecontrolv1.Entity, error) {
+	organizations, err := m.github.ListOrganizations(ctx, "")
```

Comment:
we should just make `""` a constant kinda like what kubernetes has, when you reference all namespaces its `""` but they have a type [metav1.NamespaceAll](https://github.com/kubernetes/apimachinery/blob/da970ee67a0683f95ab94dece488e7adb4eaacdf/pkg/apis/meta/v1/types.go#L288) so its clear to the reader why its an empty string.